index.ts

```diff
@@ -116,16 +117,17 @@ function function_<
 	TArgs extends Parameters<TFunction>
 >(
 	getter: TFunction,
-	{cacheKey, maxAge = 30, staleWhileRevalidate = 0, shouldRevalidate}: MemoizedFunctionOptions<TArgs, TValue> = {}
+	{cacheKey, maxAge = {days: 30}, staleWhileRevalidate = {days: 0}, shouldRevalidate}: MemoizedFunctionOptions<TArgs, TValue> = {}
 ): TFunction {
 	const getSet = async (key: string, args: TArgs): Promise<TValue | undefined> => {
 		const freshValue = await getter(...args);
 		if (freshValue === undefined) {
 			await delete_(key);
 			return;
 		}
+		const milliseconds = toMilliseconds(maxAge) + toMilliseconds(staleWhileRevalidate)
```

Comment:
😅 This may be unsafe because it can be larger than `Number.MAX_SAFE_INTEGER` and overflow

Reply:
Nah, we're very safely within the bounds for millions of years